src/installer/tests/Assets/TestProjects/ManagedHost/ManagedHost.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
+    <RuntimeFrameworkVersion>$(MNAVersion)</RuntimeFrameworkVersion>
```

Comment:
What is `MNAVersion`?

Reply:
`MicrosoftNetCoreAppVersion` (I think that's what it stands for) - it's the version of the `Microsoft.NETCore.App` framework which is locally built. Currently this should be `7.0.0`.